### PR TITLE
[4.0] Alpha sort site lang strings part 1

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes
as title says
from com_ajax.ini to mod_articles_categories.ini

Also sorted and normalised site lib.joomla.ini and admin lib.joomla.ini

### CODE REVIEW

**Comment by brianteeman** [comment]
The sorting looks fine. 

When you say "normalise" could you explain please so it can be tested

**Comment by infograf768** [comment]
> When you say "normalise" could you explain please so it can be tested

i.e. site lib.joomla.ini is now exactly the same as the admin one, as should be.

**Comment by brianteeman** [comment]
thanks for the explanation

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 5aec562df5033c9b1fa831157eb93c263f0fc45e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29605">issues.joomla.org/tracker/joomla-cms/29605</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 5aec562df5033c9b1fa831157eb93c263f0fc45e<br /><br />Looks good by review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29605">issues.joomla.org/tracker/joomla-cms/29605</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29605">issues.joomla.org/tracker/joomla-cms/29605</a>.</sub>
